CDP4ReferenceDataMapper/Managers/DataSourceManager.cs

```diff
@@ -278,7 +278,7 @@ private void UpdateColumns()
                 this.DataTable.Columns.Add(TypeColumnName, typeof(string));
             }
 
-            var newActualFiniteStates = this.actualFiniteStateList.ActualState.Except(this.Columns.OfType<ActualStateDataGridColumn>().Select(x => x.ActualFiniteState)).ToList();
+            var newActualFiniteStates = this.actualFiniteStateList.ActualState.Except(this.Columns.OfType<ActualStateDataGridColumn>().Select(x => x.ActualFiniteState)).OrderBy(x => x.ShortName).ToList();
```

Comment:
there is an ordering defined for actual finite states which i think is implemented in `CDP4Common.Comparers.ActualFiniteStateComparer`